packages/vetrina/src/Vetrina/Purchase/NewPurchase.purs

```diff
@@ -64,7 +69,13 @@ component logger = do
       , productSelection: props.productSelection <|> head props.products
       , paymentMethod: props.paymentMethod
       , showProductContents: false
+      , invalidPassword: invalidPassword
```

Comment:
This could use short hand `, invalidPassword`.

Reply:
Thanks! ^_^